UIComponents/Sources/UIComponents/Components/Texts/ParagraphView.swift

```diff
@@ -0,0 +1,50 @@
+import UIKit
+
+public extension ParagraphView {
+    struct Model {
+        public let title: String
+
+        public init(
+            title: String
+        ) {
+            self.title = title
+        }
+    }
+}
+
+public final class ParagraphView: UIView {
+    // MARK: - Private properties
+    private lazy var titleView: UITextView = {
+        let view = UITextView().prepareForAutoLayout()
+        view.textColor = designEngine.colors.textPrimary.uiColor
+        view.font = designEngine.fonts.primary.light(14).uiFont
+        view.textAlignment = .left
+        view.sizeToFit()
```

Comment:
Also style approach can be used:
view.apply(style) 
where style contains color, font, alignment, etc..